src/LazyImage/README.md

```diff
@@ -5,8 +5,8 @@ image loading performance. It is part of [the Symfony UX initiative](https://sym
 
 It provides two key features:
 
-* a Stimulus controller to load lazily heavy images, with a placeholder
-* a [BlurHash implementation](https://blurha.sh/) to create data-uri thumbnails for images
+-   a Stimulus controller to load lazily heavy images, with a placeholder
```

Comment:
why extra spaces ?

Reply:
Because we are using a tab width of 4 spaces, see https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAqABAQ3QZRgSwFsBXAG2IGd1JYAnCU0uW9GCdUiTAEw8wC98pAJ7oAFnEwA3UUUwBzOBQA06AO74YYrOgAOpTGDhiG3ZgB0oGbAG0AQuVoAJTBW1F9cQghiYC0AF0ACjEYGF0KJAB6KIAjRzFMADo3KIBKVnYwWkl4dG4-TABaYlp8VjFiQlioTCEqADMIFjlFChBlEAhdfyh2pFBMWno1AAUhhH6QTFI1TGF2ztjaQwBrOBgcXUN8KHlkGFpiOE6vWLhuM24AGUw94gU4ADFmwj8CPeRp4jYOkFDCKQAOpiTRKbZGHCTTT4KSaYRfMAURYgXYUZgwUYreRvZANGbozoAKwoAA87CswOtNphvNddnA8QSTiASaScLt5EwAIrECDwJmkQkgba0dG0L5vWirbgQNRQP66MqwIH4bhaZAADgADJ0lRB0UCVrovkqlMwpIzOgBHPnwLHdKauIpQOAXC5-HK2-A5LEKXFIfFClnowj4A5HEOcnl2xmB5mdXyxVXqsTIAAsiZWQk5AGEIIQAyAlABWP6UOAAFUwsX6QeFUmOAEkoGZYDhsvgegBBVt4YRMQXogC+w6AA